model/dao/UserDAO/HybridUserDAO.php

```diff
@@ -417,35 +418,14 @@ public function getJourneyHistory($userId) {
     /** User updater for LDAP/PostgreSQL Hybrid.
      *
      * This function updates the data of a User by its {@link UserVO}.
+     * WARNING: this function allows to update to a name that does not exist in LDAP.
+     * Do we really want this behavior?
```

Comment:
Do we do something about this? This could make sense as a transient state before we change the user in LDAP, and in this state the user would not be able to login. Alternatively, we could check that we rename to another login that exists in LDAP so we would have to modify LDAP first. In any case, this should be a very rarely used operation.

Reply:
I agree that this is such a rare case that we can leave it like this for now.

Reply:
Thanks, I think I'll rewrite the warning message to state the consequences clearly instead of asking this open question.